xmlbuilder/xmlbuilder.d.ts

```diff
@@ -4,86 +4,83 @@
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 declare module 'xmlbuilder' {
-    export = xmlbuilder;
-    class XMLDocType {
-        clone(): XMLDocType;
-        element(name: string, value?: Object): XMLDocType;
-        attList(elementName: string, attributeName: string, attributeType: string, defaultValueType?: string, defaultValue?: any): XMLDocType;
-        entity(name: string, value: any): XMLDocType;
-        pEntity(name: string, value: any): XMLDocType;
-        notation(name: string, value: any): XMLDocType;
-        cdata(value: string): XMLDocType;
-        comment(value: string): XMLDocType;
-        instruction(target: string, value: any): XMLDocType;
-        root(): XMLDocType;
-        document(): any;
-        toString(options?: Object, level?: Number): string;
+  export class XMLDocType {
```

Comment:
same here, there is no need for all these whitespace changes; it results in a bad diff / history tracking.